crates/sui-benchmark/src/bin/stress.rs

```diff
@@ -631,7 +631,7 @@ async fn main() -> Result<()> {
         (
             primary_gas_id,
             primary_gas_account,
-            Arc::new(keypair.parse()?),
+            Arc::new(keypair.parse().map_err(|e| anyhow!("{:#?}", e))?),
```

Comment:
narwhal interface change to eyre, we should replace all anyhow with eyre in sui as well